crates/eth-rpc/src/api/net_api.rs

```diff
@@ -0,0 +1,12 @@
+use jsonrpsee::core::RpcResult as Result;
+use jsonrpsee::proc_macros::rpc;
+use reth_primitives::U64;
+
+// TODO: Define and implement of methods of Net API
```

Comment:
what are the other API for this namespace? could we possibly add `todo!()` and then create issues, and then let contrib work on them ?

Reply:
make sense, I have added todo!().